roles/openshift_node_group/files/sync.yaml

```diff
@@ -119,16 +119,16 @@ spec:
 
             # does the openshift-ca.crt exist
             if [[ -f /etc/pki/ca-trust/source/anchors/openshift-ca.crt ]]; then
-              md5sum /etc/pki/ca-trust/source/anchors/openshift-ca.crt > /tmp/.old-openshift-ca
+              openssl x509 -noout -fingerprint -sha256 -inform pem -in /etc/pki/ca-trust/source/anchors/openshift-ca.crt > /tmp/.old-openshift-ca
```

Comment:
does some metadata change periodically independent of the actual certificate contents? I'm trying to understand why simple file comparison of /etc/pki/ca-trust/source/anchors/openshift-ca.crt and /run/secrets/kubernetes.io/serviceaccount/ca.crt is insufficient.

Reply:
@sdodson nope.  And I think you are right that is a better way to go.